deploy/docker-compose.yaml

```diff
@@ -0,0 +1,18 @@
+version: "3"
+services:
+  deploy:
+    image: allofustest/workbench:buildimage-0.0.13
+    entrypoint: /bootstrap-docker.sh
+    user: circleci
+    environment:
+      # Required: a GitHub release git tag, e.g. v1-0-rc1
+      - WORKBENCH_VERSION
```

Comment:
nit: DATA_BROWSER_VERSION here and above

Reply:
Done.